docs/source/model_design/vivarium_overview/vivarium_versus_other_model_types/index.rst

```diff
@@ -48,12 +49,21 @@ Microsimulation models incorporate relevant characteristics, including:
 - Adverse events
 - Mortality rate/life expectancy 
 
-.. image:: microsim_example_table_2.PNG
+The below tables provide a simplified illustration of what is included in a Vivarium microsimulation. Both tables are based on a microsimulation of 
+antenatal nutritional supplementation effects on birthweight and child-growth failure. 
 
-The above table provides an illustration of the simulant attributes that might be included in a Vivarium microsimulation. This particular example is pulled from a microsimulation of 
-antenatal nutritional supplementation effects on birthweight and child-growth failure. As you can see, the simulant listed here has attributes covering demography, risk factors (e.g., 
-*did the mother/birthing person received iron and folic acid (IFA) supplementation during pregnancy?*), causes (e.g., *what is the probability of child simulant having protein energy malnutrition (PEM)?*),
-and mortality (e.g., *is the simulant alive or dead at this given timestep?*).
+.. image:: microsim_example_table_3.PNG
```

Comment:
In addition to these images, can you include the Excel files you used to create these? That will make it much easier for people to edit in the future.

Reply:
Oh good point! Will do